src/controller/cmp.ts

```diff
@@ -14,7 +14,7 @@ export const cmpController = async (req: Request, res: Response) => {
   res.setHeader("Content-Type", "application/javascript");
```

Comment:
This will mess up the history of these metrics.
I think we can leave with it if it totally necessary as the related dashboard is not use frequently, but if there is a chance to keep the channel name here, it would be beneficial.